### ISSUE TITLE
A way to get data of an offline user

### ISSUE THREAD

**Issue by Kampfkarren** [Kampfkarren/Roblox#44]
Perhaps `DataStore2.FromUserId`?

**Comment by Dog2puppy** [comment]
Will this be added any time soon?

**Comment by Kampfkarren** [comment]
I've been working on a lot of projects other than DataStore2, so probably not from me.

**Comment by Dog2puppy** [comment]
Ok. I'll attempt to add it sometime in the near future and send in a PR.

**Comment by DeepBlueNoSpace** [comment]
@Dog2puppy Have you made any progress on this?

**Comment by Dog2puppy** [comment]
Nope. I completely forgot about this. Not sure when I'll try implementing it. I've been busy with school.

**Comment by LucasMZReal** [comment]
I might be able to add it, not sure.

**Comment by LucasMZReal** [comment]
What about integrating it into the ds2.__call() call?
Then just do it differently, like needing to clean up the cache with a function. Maybe a function / parameter to check if it's offline player data?

**Comment by Kampfkarren** [comment]
There's a lot of gotchas that come with offline data that make me not want to pack it on top of the main method. Saving only works if they're not in game. If you save while the player is in game, then when they leave your data will be overridden.

**Comment by boatbomber** [comment]
Is there any update on this? I have use cases that need some data from offline users and I would love to be able to use DS2 for it to keep my codebase consistent.
